src/App.jsx

```diff
@@ -110,9 +111,10 @@ function App() {
       <Box className={classes.page}>
         <Box className={classes.contentContainer}>
           {activeStep === 0 && <ScoringInfoPage />}
-          {activeStep === 1 && <ProductBarInfoPage />}
-          {activeStep === 2 && <CatalogueInfoPage />}
-          {activeStep === 3 && <FinishPage />}
+          {activeStep === 1 && <SourceInfoPage />}
+          {activeStep === 2 && <ProductBarInfoPage />}
+          {activeStep === 3 && <CatalogueInfoPage />}
+          {activeStep === 4 && <FinishPage />}
```

Comment:
The `handleSkip` function (line 101 above) needs to be updated as well. It should set the `activeStep` to be 4.